api/package-lock.json

```diff
@@ -16745,9 +16745,9 @@
             }
         },
         "node_modules/nodemailer": {
-            "version": "6.9.5",
-            "resolved": "https://registry.npmjs.org/nodemailer/-/nodemailer-6.9.5.tgz",
-            "integrity": "sha512-/dmdWo62XjumuLc5+AYQZeiRj+PRR8y8qKtFCOyuOl1k/hckZd8durUUHs/ucKx6/8kN+wFxqKJlQ/LK/qR5FA==",
+            "version": "6.9.9",
```

Comment:
Is this a nested dependency? Notice you've updated your lock file to latest, but your package.json file is still on the older version. Snyk shows the dependency as direct, not nested 
![image](https://github.com/JiscSD/octopus/assets/104978640/a5cc136c-07a8-42c3-9818-43b4d899c8de)

Reply:
Yes it's direct, but I just ran "npm update nodemailer", so the semver specified in package.json doesn't change, but it fetches the latest version that matches the range and updates package-lock.json.

Reply:
Odd it doesn't do both. Would recommend you manually update main package file then. If you need to reclone this or lock file goes missing for whatever reason it will install the older/insecure version.

Check out https://www.npmjs.com/package/npm-check-updates, very useful, updates both files and has a nice interactive CLI to work with.

Reply:
Sure, just updated package.json. I used npm-check-updates a while ago and was trying to remember what it was called! Thanks